packages/calypso-build/webpack.config.js

```diff
@@ -54,6 +54,7 @@ function getWebpackConfig(
 		'output-path': outputPath = path.join( process.cwd(), 'dist' ),
 		'output-filename': outputFilename = '[name].js',
 		'output-library-target': outputLibraryTarget = 'window',
+		'output-jsonp-function': outputJsonpFunction = 'webpackJsonp',
```

Comment:
Maybe also add a line to the docblock comment few lines above? The existing documentation there is not terribly helpful though 🙁  (`output-path` sets output path and `output-library-target` sets output library target, who would have guessed that)

Reply:
👍 Done in https://github.com/Automattic/wp-calypso/pull/46252/commits/b42eaaa94c94f7cf28832b4fccbf0f71875a002b